src/py_strapi/export/importer.py

```
@@ -0,0 +1,545 @@
+                        # Track ID mapping for relation resolution
+                        if content_type not in result.id_mapping:
+                            result.id_mapping[content_type] = {}
+
+                        result.id_mapping[content_type][entity.id] = response.data.id
+                        result.entities_imported += 1
+
+                except ValidationError as e:
+                    result.add_error(f"Validation error importing {content_type} #{entity.id}: {e}")
+                    result.entities_failed += 1
+
+                except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)